src/SMCDEL/Internal/TexDisplay.hs

```diff
@@ -48,7 +49,8 @@ class TexAble a where
     ts <- round <$> getPOSIXTime
     let filename = tmpdir ++ "/disp-" ++ show (ts :: Int)
     pdfTo x filename
-    runIgnoreAndWait $ "/usr/bin/okular " ++ filename ++ ".pdf"
+    runIgnoreAndWait $ "/usr/bin/xdg-open " ++ filename ++ ".pdf"
+    threadDelay 3000000 -- give viewer three seconds before deleting tmpdir
```

Comment:
Workaround for the problem discussed at https://stackoverflow.com/questions/46736478/perl-wait-for-xdg-open-to-quit-before-continuing